example/app-framework/build.sh

```diff
@@ -0,0 +1,167 @@
+#
+# Copyright (C) 2023 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+#!/bin/bash
+
+CURR_DIR=$PWD
+WAMR_DIR=${PWD}/../../runtime-library/deps/wamr-gc
+SIMPLE_DIR=${WAMR_DIR}/samples/simple
+OUT_DIR=${PWD}/out
+BUILD_DIR=${PWD}/build
+
+IWASM_ROOT=${WAMR_DIR}/core/iwasm
+APP_FRAMEWORK_DIR=${WAMR_DIR}/core/app-framework
+NATIVE_LIBS=${APP_FRAMEWORK_DIR}/app-native-shared
+APP_LIB_SRC="${APP_FRAMEWORK_DIR}/base/app/*.c ${APP_FRAMEWORK_DIR}/sensor/app/*.c \
+             ${APP_FRAMEWORK_DIR}/connection/app/*.c ${NATIVE_LIBS}/*.c"
+WASM_APPS=${SIMPLE_DIR}/wasm-apps
+CLEAN=
+CM_BUILD_TYPE="-DCMAKE_BUILD_TYPE=Release"
+CM_TOOLCHAIN=""
+
+usage ()
+{
+    echo "build.sh [options]"
+    echo " -p [profile]"
+    echo " -d [target]"
+    echo " -c, rebuild SDK"
+    exit 1
+}
+
+
+while getopts "p:dch" opt
+do
+    case $opt in
+        p)
+        PROFILE=$OPTARG
+        ;;
+        d)
+        CM_BUILD_TYPE="-DCMAKE_BUILD_TYPE=Debug"
+        ;;
+        c)
+        CLEAN="TRUE"
+        ;;
+        h)
+        usage
+        exit 1;
+        ;;
+        ?)
+        echo "Unknown arg: $arg"
+        usage
+        exit 1
+        ;;
+    esac
+done
+
+
+if [ "$CLEAN" = "TRUE" ]; then
+    rm -rf $CURR_DIR/cmake-build
+fi
+
+
+while  [ ! -n "$PROFILE" ]
+do
+    support_profiles=`ls -l "${SIMPLE_DIR}/profiles/" |grep '^d' | awk '{print $9}'`
```

Comment:
Don't use original profile files in WAMR repo, should pointing to our own profiles